File: src/main/kotlin/ympa_kotlin_wiremock_server/models/PartnerShipmentWarehouseDTO.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class PartnerShipmentWarehouseDTO(
    @field:JsonProperty("id")
    val id: kotlin.Long? = null,

    @field:JsonProperty("name")
    val name: kotlin.String? = null,

    @field:JsonProperty("address")
    val address: kotlin.String? = null,

)

File: src/App/DTO/PartnerShipmentWarehouseDTO.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Данные о складе отправления.
 */
class PartnerShipmentWarehouseDTO
{
    /**
     * Идентификатор склада отправления.
     * @DTA\Data(field="id", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"int"})
     */
    public ?int $id = null;

    /**
     * Наименование склада отправления.
     * @DTA\Data(field="name", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $name = null;

    /**
     * Адрес склада отправления.
     * @DTA\Data(field="address", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $address = null;

}
